### PR TITLE
Show a warning dot with number of insufficient fund orders

### PR DESCRIPTION
**Pull request by hungdoansy**
| Before | After |
| --- | --- |
| image | image |

- [x] Confirm auto checked bug
- [x] Revert env settings

### CODE REVIEW

File: src/components/swapv2/LimitOrder/request.ts [hunk 1]
```diff
@@ -23,6 +23,18 @@ export const getListOrder = (
...
+export const getNumberOfInsufficientFundOrders = async (
+  params: {
+    chainId: ChainId
+    maker: string
+  },
+  signal?: AbortSignal,
+): Promise<number> => {
+  return fetch(`${LIMIT_ORDER_API_READ}/v1/orders/insufficient-funds?${stringify(params)}`, { signal })
+    .then(data => data.json())
+    .then(data => data.data.total || 0)
```
Comment (nguyenhoaidanh) [inline src/components/swapv2/LimitOrder/request.ts:35]: data?.data?.total for sure

File: .env [hunk 2]
```diff
@@ -1,40 +1,38 @@
...
+REACT_APP_LIMIT_ORDER_API_WRITE=https://limit-order.stg.kyberengineering.io/write/api
+REACT_APP_LIMIT_ORDER_API_READ=https://limit-order.stg.kyberengineering.io/read-ks/api
 
 # Kyber DAO Apis
-REACT_APP_KYBER_DAO_STATS_API=https://kyberswap-dao-stats.kyberengineering.io
-
-REACT_APP_PRICE_ALERT_API=https://price-alert.dev.kyberengineering.io/api
-
+REACT_APP_KYBER_DAO_STATS_API=https://kyberswap-dao-stats.stg.kyberengineering.io
+REACT_APP_PRICE_ALERT_API=https://price-alert.stg.kyberengineering.io/api
```
Comment (nguyenhoaidanh) [inline .env:35]: todo: let's revert after testing finished

**Review by nguyenhoaidanh — APPROVED** [review]

File: .env (hunk 2 above)
Comment (hungdoansy) [reply .env:35]: Will do bro

Comment (viet-nv) [reply .env:35]: should revert now?

**Review by viet-nv — DISMISSED** [review]

**Pull request merged**